### DISCUSSION TITLE
Really impressive!

### DISCUSSION THREAD

**Discussion by Aap-Dev** [VendelinSlezak/BleskOS#13] [Show and tell]
I also just started a assembly only OS project and was wondering if you could give any advice that would be appreciated! 
Keep going it's really good!

**Comment by VendelinSlezak** [comment]
Hello, osdev is very nice and interesting hobby, so my advice would be also keep going 🙂 If you will need help with something, just ask me and I will try to do my best.
BTW I am glad that you like my code and you copied it into your project, but please insert to every file line like ; this file contains code from BleskOS github.com/Klaykap/BleskOS or something similar

**Comment by ghost** [comment]
I've been following this since the beginning and I must say it has advanced since then 🙃

**Comment by VendelinSlezak** [comment]
And now I can say that another great advance is near. I am working on network connection, and yesterday was first time when I successfully transferred HTML file from server and show it in BleskOS internet browser. So now I have to complete drivers for other cards(now only AMD PCnet works on 100% but others are very near), make few improvements of TCP stack and update internet browser rendering of HTML. I hope I will upload this full big update next week. I already push some code, but it is only for backup purpose and it is not connected with whole code. So keep watching 🙂
